nexus/types/src/deployment.rs

```diff
@@ -1225,6 +1227,7 @@ impl From<&crate::inventory::Dataset> for CollectionDatasetIdentifier {
 pub struct UnstableReconfiguratorState {
     pub planning_input: PlanningInput,
     pub collections: Vec<Collection>,
+    pub target_blueprint: Option<BlueprintTarget>,
```

Comment:
I think the `Option` is fine. Maybe worth adding a comment about it though.

Reply:
:+1: added in [1cc0815](https://github.com/oxidecomputer/omicron/pull/7485/commits/1cc0815b109fa3ee46135772aebeba84800e72d3)